### ISSUE TITLE
Danger crashes after plugin update

### ISSUE THREAD

**Issue by garnett** [ashfurrow/danger-ruby-swiftlint#82]
```diff
-danger-swiftlint (0.10.1)
+danger-swiftlint (0.12.0)
```

CI run has failed with the following message:

```
[!] Invalid Dangerfile file: undefined method positive? for 0:Fixnum

 #  from /tmp/sandbox/workspace/Dangerfile:51
 #  -------------------------------------------
 #  swiftlint.config_file = '.swiftlint.yml'
 >  swiftlint.lint_files inline_mode: true
 #  
 #  -------------------------------------------
```
CI uses `ruby-2.3.1`

**Comment by ashfurrow** [comment]
Yikes, sorry about that! That is very strange. I installed `ruby-2.3.1` with RVM and with a fresh `irb` shell, it appeared to work:

```irb
2.3.1 :001 > 0.positive?
 => false
```

I'm not sure what this could be – I'll take a look tomorrow. Anyone else have ideas?

**Comment by ashfurrow** [comment]
Okay, this was fixed by @thii and released as 0.12.1. Please let us know how it goes 👍

**Comment by garnett** [comment]
Worked like a charm 🎆
